### DISCUSSION TITLE
Mongodb 5.1 on windows images?

### DISCUSSION THREAD

**Discussion by mikkelgunvald** [actions/virtual-environments#4481] [Installed software]
I saw mongodb was updated from 5.0.x to 5.1 recently on windows images. Was is intentionel?
 

> MongoDB 5.1 is a rapid release and is only supported for MongoDB Atlas. MongoDB 5.1 is not supported for use on-premises. 
https://docs.mongodb.com/v5.1/release-notes/5.1/

**Comment by miketimofeev** [comment]
Hi @mikkelgunvald!
We install the latest available 5.* version of Mongodb using chocolatey. Unfortunately, we were not aware of the changes in 5.1.
We will revert the version next week.

**Reply by mikkelgunvald** [reply]
Hi @miketimofeev 
Thx for the fast reply and info :)
